integration-tests/test/helloWorld.js

```diff
@@ -12,7 +13,8 @@ describe("Hello World", () => {
       { type: "SYSTEM_OUT", value: "\n" },
       { type: "STATUS", value: "EXITED" },
     ];
+    const assertOnMessagesReceived = receivedMessages => expect(expectedMessages).to.deep.equal(receivedMessages);
```

Comment:
nit: should this be moved into `testHelpers` since it will most likely be used by other integration tests?

Reply:
Yeah that sounds good, I'm kind of moving stuff around as I'm adding more tests, so I'll do that in my next integration test PR.